### DISCUSSION TITLE
v0.9.2

### DISCUSSION THREAD

**Discussion by iioter** [iioter/iotgateway#27] [Announcements]
- 从v0.9.2开始，默认移除OPCUA Server，你可以手动开启
- 设备编辑禁止修改驱动

<hr /><em>This discussion was created from the release <a href='https://github.com/iioter/iotgateway/releases/tag/v0.9.2'>v0.9.2</a>.</em>

**Comment by gyp8735abc** [comment]
大佬，我想使用iotgateway推送数据到thingsboard，那在iotgateway中只能使用mqtt-client，但和iotgateway在一起的设备也想获取设备相关的参数值，使用http://iotgateway.net/docs/iotgateway/mqtt的主题没有获得消息，或者我的配置不对（文档写的不太明白），希望指点，求教了，各位大佬。

**Reply by iioter** [reply]
你是说网关所连接的从设备，需要获取tb的共享属性或者客户端属性是吗？

**Reply by gyp8735abc** [reply]
我本地有写的其他程序，想读取tb里面的数据，如何实现？
比如本地有别的上位机，或者HMI。

**Reply by gyp8735abc** [reply]
可不可以，即使通过mqtt-client上传数据，也抄送一份数据到本地的mqtt-server中，这样就可以共享数据了。
后续希望可以有设备影子的功能，希望大佬考虑。

**Reply by iioter** [reply]
进群讨论吧 QQ微信都有 扫码加入

**Comment by gyp8735abc** [comment]
![image](https://user-images.githubusercontent.com/13816018/184297187-d17eaac7-a5e1-462e-9fd3-12c5b7ac89a8.png)
大佬，这个属性上报怎么配置，没有找到介绍或者操作文档，希望指导，谢谢。
